Automattic/wp-calypso
# Home: Reorganize components in locations and cards

#### Changes proposed in this Pull Request

As part of the ongoing work seeking to speed up creation of My Home views after site setup, this PR:
* Reorganizes the components rendered in My Home in 2 main folders: one for the layout locations (notices, upsells, primary and secondary) and another one for all the cards (which are self-organized in card types: education, features, notices, primary).
* Moves the existing locations and cards which have been moved to their own components (namely the notices location and the celebrate, free photo library, go mobile, stats and quick links cards) into the new folders.
* Moves the primary location and the site setup checklist to their own components.

<img width="327" alt="Screen Shot 2020-03-23 at 16 17 51" src="https://user-images.githubusercontent.com/1233880/77332088-def26f80-6d21-11ea-8ea1-51e9c184459b.png">

Rest of the locations and cards will be covered in a follow-up PR in order to keep this one small.

#### Testing instructions

- Visit "My Home".
- Make sure there are no regressions. In particular:
  - Established sites keep getting the quick links.
  - Unestablished sited keep getting the checklist.
  - There are no styling changes in the celebrate, free photo library, go mobile and stats cards.
